Marlin/src/pins/stm32f4/pins_LERDGE_K.h

```diff
@@ -213,17 +233,21 @@
 // LCD / Controller
 //
 
-#define TFT_RESET_PIN                       PD6
-#define TFT_BACKLIGHT_PIN                   PD3
+#if HAS_FSMC_TFT
+  #define TFT_RESET_PIN                     PD6
+  #define TFT_BACKLIGHT_PIN                 PD3
 
-#define TFT_CS_PIN                          PD7
-#define TFT_RS_PIN                          PD11
+  #define TFT_CS_PIN                        PD7
+  #define TFT_RS_PIN                        PD11
 
-#define TOUCH_CS_PIN                        PG15
-#define TOUCH_SCK_PIN                       PB3
-#define TOUCH_MOSI_PIN                      PB5
-#define TOUCH_MISO_PIN                      PB4
+  #define TOUCH_CS_PIN                      PG15
+  #define TOUCH_SCK_PIN                     PB3
+  #define TOUCH_MOSI_PIN                    PB5
+  #define TOUCH_MISO_PIN                    PB4
+#endif
 
-#define BTN_EN1                             PG10
-#define BTN_EN2                             PG11
-#define BTN_ENC                             PG9
+#if IS_NEWPANEL
+  #define BTN_EN1                           PG10
+  #define BTN_EN2                           PG11
+  #define BTN_ENC                           PG9
+#endif
```

Comment:
I do not know how to do this correctly (maybe somewhere in `Configuration.h`/`Configuration_adv. h`).
Here is the display from Lerdge. It has a built-in encoder and beeper.
![Lerdge Display V1 0 0](https://user-images.githubusercontent.com/42996016/117411258-0f906b00-af1c-11eb-8109-3e5fceee9de1.jpg)
![Lerdge Display V1 0 0 bottom](https://user-images.githubusercontent.com/42996016/117411277-13bc8880-af1c-11eb-811f-d958f4dea023.jpg)

Reply:
We'll try to work it out on #21815.